### PR TITLE
[Feature] Finalized Texts and UI polish for ExposureNotificationSetting screen

### PR DESCRIPTION
**Pull request by llanghammer-sap**
Updated the latest german texts for the ExposureNotificationSetting screen and added the stubs for english translation later on. 

A few minor design changes were added to follow the design.

You can see the changes in the screenshots below:

IMG_0351 

IMG_08A109B3E990-1

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift [hunk 1]
```diff
@@ -22,10 +22,18 @@ class DescriptionTableViewCell: UITableViewCell, ConfigurableENSettingCell {
...
 
-	func configure(for _: RiskDetectionState) {
-		titleLabel.text = AppStrings.ExposureNotificationSetting.descriptionTitle
+	func configure(for riskDetectionState: RiskDetectionState) {
+		if riskDetectionState == .disabled {
+			titleLabel.text = AppStrings.ExposureNotificationSetting.descriptionTitleInactive
+		} else {
+			titleLabel.text = AppStrings.ExposureNotificationSetting.descriptionTitle
+		}
+		titleLabel.font = UIFontMetrics.default.scaledFont(for: UIFont.boldSystemFont(ofSize: 22))
```
Comment (thomasaugsten) [inline src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift:33]: Maybe use preferredFont for accessibility

File: src/xcode/ENA/ENA/Source/Views/ENSetting/ActionDetailTableViewCell.swift [hunk 2]
```diff
@@ -48,6 +48,7 @@ class ActionDetailTableViewCell: UITableViewCell, ConfigurableENSettingCell {
 		iconImageView2.image = images(for: state).1
 		actionButton.setTitle(AppStrings.ExposureNotificationSetting.detailActionButtonTitle, for: .normal)
 
+		actionTitleLabel.font = UIFontMetrics.default.scaledFont(for: UIFont.boldSystemFont(ofSize: 22))
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Views/ENSetting/ActionDetailTableViewCell.swift:51]:
```suggestion
		actionTitleLabel.font = UIFont.preferredFont(forTextStyle: .title)
```

File: src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift (hunk 1 above)
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift:33]:
```suggestion
		titleLabel.font = UIFont.preferredFont(forTextStyle: .title)
```

File: src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift [hunk 3]
```diff
@@ -22,10 +22,18 @@ class DescriptionTableViewCell: UITableViewCell, ConfigurableENSettingCell {
...
 		textView1.text = AppStrings.ExposureNotificationSetting.descriptionText1
+		textView1.font = UIFontMetrics.default.scaledFont(for: UIFont.boldSystemFont(ofSize: 17))
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift:35]:
```suggestion
		textView1.font = UIFont.preferredFont(forTextStyle: .body)
```

File: src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift (hunk 1 above)
Comment (llanghammer-sap) [reply src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift]: Done with https://github.com/corona-warn-app/cwa-app-ios/pull/112/commits/4bbf3f3e49cd921879b33ccb5c710d49b6f53d95

File: src/xcode/ENA/ENA/Source/Views/ENSetting/ActionDetailTableViewCell.swift (hunk 2 above)
Comment (llanghammer-sap) [reply src/xcode/ENA/ENA/Source/Views/ENSetting/ActionDetailTableViewCell.swift]: Done with https://github.com/corona-warn-app/cwa-app-ios/pull/112/commits/4bbf3f3e49cd921879b33ccb5c710d49b6f53d95 used .title2 to get the right textsize

File: src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift (hunk 1 above)
Comment (llanghammer-sap) [reply src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift]: Done with https://github.com/corona-warn-app/cwa-app-ios/pull/112/commits/4bbf3f3e49cd921879b33ccb5c710d49b6f53d95 used .title2 to get the right textsize.

File: src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift (hunk 3 above)
Comment (llanghammer-sap) [reply src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift]: Done with https://github.com/corona-warn-app/cwa-app-ios/pull/112/commits/4bbf3f3e49cd921879b33ccb5c710d49b6f53d95 used .headline to be closer to the design

**Comment by llanghammer-sap** [comment]
@thomasaugsten @ChristianKienle I switched to the preferredFonts which differ from the designs provided by the Mockups. The scaled font for achieving **bold** texts of the preferredFonts should scale as well when adjusting the text sizes. 

If it is okay with you, I would switch back to the bold fonts while making sure that the fonts scale as specified in the accessibility settings.

**Review by ChristianKienle — APPROVED** [review]

**Review by eisvogel — APPROVED** [review]

**Pull request merged**